docs/postgres.rst

```diff
@@ -6,7 +6,19 @@ Postgres version 9.4 or later is known to work.
 Set up database
 ===============
 
-The PostgreSQL database used *must* have the correct encoding set, otherwise
+Assuming your PostgreSQL database user is called ``postgres``, create a user
+``synapse_user`` with::
+
+ su - postgres
+ createuser --pwprompt synapse_user
+ su - synapse_user
```

Comment:
this bit is now unnecessary

Reply:
this is outdated

Reply:
? it's still showing up in the diff at https://github.com/matrix-org/synapse/pull/1756/files

Reply:
Ok,so what lines have to be removed?

Reply:
done